### PR TITLE
project updated for code review.

### PR DESCRIPTION
**Pull request by DilsherB**
Pull request to review the project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -25,4 +25,4 @@ input[type=text],input[type=email],input[type=number], select, textarea, button{
 }
 button{
   background-color: #38ac64;
-}
```
Comment (DilsherB) [inline css/style.css:28]: your work is excellent but there must be one empty line at the end of CSS file. so I suggest to include one blank line at the end of file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_module
+README.md
```
Comment (DilsherB) [inline .gitignore:2]: you perfectly added .gitignore but it also needs one blank line at the end of the file.

**Review by DilsherB — COMMENTED** [review]
Overall your work is very good.

File: .gitignore (hunk 2 above)
Comment (Olamarx) [reply .gitignore:2]: > you perfectly added .gitignore but it also needs one blank line at the end of the file.

- [ ] It is always important to add the WHY, this is for the student to know the reason you are requesting changes.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @DilsherB,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Polite sentences ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly make use of github review feature as I have shown in the picture below.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DilsherB — COMMENTED** [review]
your code is perfect and i really appriciate it. Your project is approved but i suggest you to make the following changes. These are the coding conventions. you looks more professional when you follow coding conventions.
1. Add a blank line at the end of both css and .gitignore file

**Review by OyePriscilla — APPROVED** [review]
Hi @DilsherB 😄 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Good use of inline comments ✔️
Used GitHub's 'Review changes' feature from GitHub ✔️
Clear and polite suggestion ✔️

 [optional] It would be great to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly. 🧑‍🤝‍🧑

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DilsherB** [comment]
review approved

**Pull request merged**